client/src/dev_commands.rs

```diff
@@ -18,6 +18,8 @@ impl Command for DevCommand {
             Box::new(DevCommandCompile {}),
             Box::new(DevCommandPublish {}),
             Box::new(DevCommandExecute {}),
+            Box::new(DevCommandAddValidator {}),
+            Box::new(DevCommandRemoveValidator {}),
```

Comment:
We will eventually need `register_validator` if we want to add completely new validators instead of just removing an existing one, then adding it back right?

Reply:
totally! I'll add that when we're testing that feature